pkg/virtctl/virtctl.go

```diff
@@ -35,8 +35,18 @@ type App interface {
 type Options struct {
 }
 
-func (o *Options) FlagSet() *flag.FlagSet {
+func DefaultKubeconfig() string {
+	kubeConfig := os.Getenv("kubeconfig")
+	if kubeConfig == "" {
+		kubeConfig = os.Getenv("KUBECONFIG")
+	}
+	if kubeConfig == "" {
+		kubeConfig = "~/.kube/config"
```

Comment:
What would you think about using the `os/user` module to actually determine the correct home dir to use instead of "~"?

Reply:
yup, good idea. I didn't realize the home dir was something the os.User object had in golang, but it definitely does now that I'm looking at it.